src/ua/lviv/iot/algo/part1/lab1/Drone.java

```diff
@@ -0,0 +1,60 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+import lombok.*;
+
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class Drone {
+    private double currentSpeed;
+    private double currentAltitude;
+    private double batteryCapacity;
+    private double currentBatteryLevel;
+    private static Drone instance;
+
+    public static Drone getInstance() {
+        if(instance == null){
```

Comment:
```suggestion
        if (instance == null) {
```
Reformat your file please

Reply:
solved